src/sentry/integrations/gitlab/client.py

```diff
@@ -168,14 +154,17 @@ def create_issue(self, project, data):
             data=data,
         )
 
-    def search_issues(self, query):
-        return self.get(
-            GitLabApiClientPath.issues_search,
```

Comment:
Is this path still used?

Reply:
No, only `search_group_issues` is used now.

Reply:
I don't think we have a use case where we need to search every issue in a gitlab instance.